content/automate/ManageIQ/System/Event/EmsEvent/Ansible_Tower.class/_missing.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
It has to match what is defined [here](https://github.com/ManageIQ/manageiq/blob/7d7fe087cb0eea8d20ad737ff07f0a4bd7021be1/app/models/manageiq/providers/ansible_tower/automation_manager/event_parser.rb#L5).